views/home.hbs

```diff
@@ -48,7 +48,7 @@
     <div class="reveal" id="subscribe-modal" data-reveal>
       <h3>Subscribe to Breach Alerts</h3>
       <p>Explanation of what the user will get if they subscribe to breach alerts.</p>
-      <a href="#" class="button sign-in-btn" onClick="doOauth();">Subscribe with Firefox Accounts</a>
+      <a href="#" class="button sign-in-btn" onclick="doOauth();">Subscribe with Firefox Accounts</a>
```

Comment:
Is there a way to configure HTMLLint to disallow `on*` attributes? These are security no-no.

Reply:
As close as I can tell, we'd need to use the [`attr-bans`](https://github.com/htmllint/htmllint/wiki/Options#attr-bans) rule, and explicitly specify any `on*` attribute we wanted to disallow. Not great, but I don't see any other easy config or RegExp in the docs [yet].

Reply:
Oops, missed this...
Added `"onclick"` to the `attr-bans` rule... which now makes the CI fail.
Ideas on how to tweak the breakage? Or are we cool w/ merging a PR that fails the CI and we can submit a follow-up issue?

Reply:
Let's not merge a CI fail. Are you comfortable moving the script behavior into a separate `.js` file yourself, or prefer someone else do it?